README.md

```diff
@@ -0,0 +1,28 @@
+![alert
+status](https://sonarcloud.io/api/project_badges/measure?project=nightowl-studios_arcade&metric=alert_status)
+
+![coverage](https://sonarcloud.io/api/project_badges/measure?project=nightowl-studios_arcade&metric=coverage)
+
+![bugs](https://sonarcloud.io/api/project_badges/measure?project=nightowl-studios_arcade&metric=bugs)
+
+![vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=nightowl-studios_arcade&metric=vulnerabilities)
+
+![lines of
+code](https://sonarcloud.io/api/project_badges/measure?project=nightowl-studios_arcade&metric=ncloc)
+
+What is Arcade {#_what_is_arcade}
```

Comment:
This used to be `anchors` in adoc. so you can click a title and send it to someone, and they'll "anchor" to where you clicked the title. In Markdown, the syntax is different.

https://gist.github.com/asabaylus/3071099

`What is Arcade {#_what_is_arcade}` should be
`[What is Arcade](#what-is-arcade)`